backend/src/routes.ts

```diff
@@ -1,4 +1,4 @@
-import { Router } from "express";
```

Comment:
Let's keep everything consistent. Since all the imports use single quotation for packages let's stick with that.